### DISCUSSION TITLE
How to add images in select list?

### DISCUSSION THREAD

**Discussion by sh4msi** [filamentphp/filament#5566] [Help]
How to add images in select list?
Something like this photo

![download](https://user-images.githubusercontent.com/6229984/215268138-0d82b6d6-957e-43a0-8fe1-4707988aeb78.png)

**Comment by danharrin** [comment]
https://filamentphp.com/tricks/render-html-in-select-options

**Reply by sh4msi** [reply]
If I don't want it to be searchable, is there a solution?

**Reply by danharrin** [reply]
Native HTML selects don't allow for images
